source/agora/common/Serializer.d

```diff
@@ -152,11 +152,17 @@ unittest
             serializePart(this.s, dg);
         }
 
-        void deserialize (scope DeserializeDg dg) @safe
+        static QT fromBinary (QT) (
```

Comment:
We must implement this as static.
And we need to use 'deserializeFull' inside.

Reply:
https://bpfkorea.github.io/agora/agora/common/Serializer.html